src/Bicep.Core.IntegrationTests/Scenarios/InliningResourcesAndModulesTests.cs

```diff
@@ -349,7 +349,7 @@ public void AssigningModuleToVariable_ShouldNotGenerateVariables_Condition()
                     },
                     "Module should be added to depends on section");
                 result.Template.Should().HaveValueAtPath("$.resources[?(@.name == 'test2')].properties.allowBlobPublicAccess",
-                    "[if(equals(parameters('mode'), 1), reference(resourceId('Microsoft.Resources/deployments', 'testmod1'), '2020-10-01').outputs, reference(resourceId('Microsoft.Resources/deployments', 'testmod2'), '2020-10-01').outputs).test.value]",
+                    "[if(equals(parameters('mode'), 1), reference(resourceId('Microsoft.Resources/deployments', 'testmod1'), '2020-10-01'), reference(resourceId('Microsoft.Resources/deployments', 'testmod2'), '2020-10-01')).outputs.test]",
```

Comment:
Shouldn't this expression still end in `.value`?

Reply:
Great catch! Fixed.

Reply:
I'm confused. The diff doesn't show it ending in `.value`.